src/main.c

```diff
@@ -2793,6 +2784,16 @@ static h2o_iovec_t on_extra_status(void *unused, h2o_globalconf_t *_conf, h2o_re
 #undef BUFSIZE
 }
 
+static void setup_configurators_post_privsep(void)
+{
+    h2o_configurator_t *c = h2o_configurator_create(&conf.globalconf, sizeof(*c));
+
+    c->enter = on_config_listen_enter;
+    c->exit = on_config_listen_exit;
+    h2o_configurator_define_command(c, "listen", H2O_CONFIGURATOR_FLAG_GLOBAL | H2O_CONFIGURATOR_FLAG_HOST, on_config_listen);
+    h2o_fastcgi_register_configurator(&conf.globalconf);
+}
```

Comment:
I believe we do not need to change when these functions are invoked, now that the root path is provided as an argument. Correct?